src/actions/AuthActions.js

```diff
@@ -30,20 +30,25 @@ export const loginAuthError = (dispatch, err) => {
     });
 };
 
-export const loginSuccess = (dispatch) => {
+export const loginSuccess = (dispatch, token) => {
+    // TODO token arg will be used when we add FB Auth
+    let tokenToSend = token;
+
+    if (!tokenToSend) {
+        tokenToSend = 'Manually Generated Token'
+    }
+
     dispatch({
-        type: LOGIN_SUCCESS
+        type: LOGIN_SUCCESS,
+        payload: tokenToSend
     });
-
-    Actions.hidden({type: 'reset'});
-    Actions.tabbar();
 };
 
 export const loginUser = ({email, password}) => {
     return (dispatch) => {
         firebase.auth().signInWithEmailAndPassword(_.trim(email), _.trim(password))
             .then(user => {
-                loginSuccess(dispatch)
+                loginSuccess(dispatch);
```

Comment:
Removed the calls to `Action()` which is a `react-native-router-flux` call. This still needs to be fully refactored to `react-navigation` but works as-is.